src/zenml/orchestrators/local/local_orchestrator.py

```diff
@@ -41,4 +50,55 @@ def run_pipeline(
     ) -> Any:
         """Runs a pipeline locally."""
         tfx_pipeline = create_tfx_pipeline(pipeline, stack=stack)
-        LocalDagRunner().run(tfx_pipeline, run_name)
+
```

Comment:
The `LocalDagRunner` is unused after this change right? I think we can safely delete it then (if I somehow missed the deleted file in my review just ignore this)

Reply:
I actually removed the `LocalDagRunner` altogether however due to the recent changes in the `develop` branch I had to redo some of the changes. This should be fixed right now.